x-pack/legacy/plugins/uptime/public/components/functional/monitor_page_link.tsx

```diff
@@ -20,13 +20,14 @@ export const MonitorPageLink: FunctionComponent<DetailPageLinkProps> = ({
   location,
   linkParameters,
 }) => (
+  // encode monitorid param as 64 base string to make it a valid URL, since it can be a url
```

Comment:
This is kind of a nitpick, but could we put this comment closer to where the actual operation is taking place? Additional comment below.